### PR TITLE
It1 update education model

### LINKED ISSUE #39: Add attribute 'order' in education model
FE request GET /education or individual GET /education/<id>

The attributes section (attributes section) of each education object should now have a new field for creating order in the survey questions

Example(s):
`education.object.order = Integer`
`education.zipcode.order = 1`

### LINKED ISSUE #64: Create mocked data with EDUCATION return for ordering of questions built into attributes

### LINKED ISSUE #77: Add attribute 'symbol' in education model
FE request GET /education or individual GET /education/<id>

The attributes section (attributes section) of each education object should now have a new field for the symbol associated with user input for the survey question

Example(s):
`education.object.symbol = String`
`education.zipcode.symbol = '$'`

### PR DESCRIPTION
**Pull request by priyapower**
# This PR:
---

## Changes
- Updates Education model for `order` and `symbol` attributes
- Also Updates Education controller for GET by ID and All that will allow the JSON return to have some sense of ordering
- **NEEDS TEST UPDATES (see Issue #78)**
- [X] Adds code for functionality
- [ ] Adds a test
- [ ] Updates Documentation/ReadMe
- [ ] Adds dependencies
- [ ] Refactors code
- [ ] Fixes a bug

## Checklist
**Please check off what you can for this PR:**
- [ ] This code passes all tests on local machine
- [ ] This code is DRY
- [X] This code follows OOP
- [X] This code, if updates to functionality were made, updates all corresponding files

**The "I have" section is a double check for PR specific information:**
- [X] I have tagged reviewers for this PR
- [X] I have checked what branch I am merging into (current expectation is `it1_development`)
- [X] I have added this PR to the project
- [x] I have added this PR to the correct milestone
- [ ] I have updated documenation if changes were made that affect setup/running

## Linked tickest / issues 
Closes #39 
Closes #64 
Closes #77

### CODE REVIEW

**Review by Arique1104 — COMMENTED** [review]
Great job!

File: api/controller/EducationController.py [hunk 1]
```diff
@@ -12,15 +12,17 @@ def get_education(self, id):
...
-                        "classification": education.classification,
-                        "question": education.question,
-                        "description": education.description,
-                        "information": education.information,
-                        "note": education.note,
-                        "source": education.source
+                    "01_type": str(education),
+                    "02_id": str(education.id),
+                    "03_attributes": {
+                        "A_order": education.order,
```
Comment (Atos20) [inline api/controller/EducationController.py:18]: I love the letter at the beginning of each key. what was the reasoning behind having the letter?

**Review by Atos20 — COMMENTED** [review]
Great Job team!!

File: api/controller/EducationController.py (hunk 1 above)
Comment (priyapower) [reply api/controller/EducationController.py:18]: @Atos20 This helped us force ordering. We first attempted to use Python's native `OrderedDict` and we tried to use the JSON native options, but both were unsuccessful so this was our current workaround. We are looking into going back to finding a way to natively Order this dictionary, but it is a lot of detailed work that we worried we wouldn't have enough time to accomplish this and roll out a successful production server for FE

Basically - it's currently a bandaid, but it's our best bandaid, and we can explain this more during our standup this morning!
